sdk/eventhub/event-hubs/src/eventHubProducerClient.ts

```diff
@@ -139,6 +139,16 @@ export class EventHubProducerClient {
     this._producersMap = new Map();
   }
 
+  /**
+   * Creates an instance of `EventDataBatch` to which one can add events until the maximum supported size is reached.
+   * The batch can be passed to the `send()` method of the `EventHubProducer` to be sent to Azure Event Hubs.
+   * @param options  A set of options to configure the behavior of the batch.
+   * - `partitionKey`  : A value that is hashed to produce a partition assignment.
+   * Not applicable if the `EventHubProducer` was created using a `partitionId`.
+   * - `maxSizeInBytes`: The upper limit for the size of batch. The `tryAdd` function will return `false` after this limit is reached.
+   * - `abortSignal`   : A signal the request to cancel the send operation.
+   * @returns Promise<EventDataBatch>
+   */
   async createBatch(options?: BatchOptions): Promise<EventDataBatch> {
```

Comment:
Consider renaming `BatchOptions` to `CreateBatchOptions`. That brings about a symmetry between this and the `SendBatchOptions`

This change would also be in sync with the way we have been naming "options" in Keyvault & Storage where the name of the method is used as the prefix for "options"

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/41c6135c17c082f6d07fc8d90392f92ab856224d